File: Components/Document Server/CS/DocsMainDemo/Modules/SpreadsheetEmployeeInformation.cs
using System.Data;
using System.Drawing;
using System.Globalization;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.Docs.Demos;
using DevExpress.Spreadsheet;
using DevExpress.Spreadsheet.Demos;

namespace DevExpress.Docs.Demos {
    public partial class SpreadsheetEmployeeInformationModule : SpreadsheetTutorialControl {
        public SpreadsheetEmployeeInformationModule() {
            InitializeComponent();
            RefreshPreview();
            SpreadsheetTutorialPanel.Dock = DockStyle.Fill;
        }

        protected override string DefaultFileName { get { return "EmployeeInformation"; } }

        public override IWorkbook CreateBook() {
            InitializePreviewControl();
            EmployeeInformationDocumentGenerator generator = new EmployeeInformationDocumentGenerator(DefaultCulture);
            return generator.Generate(DemoUtils.GetRelativePath("EmployeeInformation_template.xlsx"));
        }

        void InitializePreviewControl() {
            SpreadsheetTutorialPanel.Location = Point.Empty;
        }
    }
}

File: Components/Document Server/VB/DocsMainDemo/Modules/SpreadsheetEmployeeInformation.vb
Imports Microsoft.VisualBasic
Imports System.Data
Imports System.Drawing
Imports System.Globalization
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.Docs.Demos
Imports DevExpress.Spreadsheet
Imports DevExpress.Spreadsheet.Demos

Namespace DevExpress.Docs.Demos
	Partial Public Class SpreadsheetEmployeeInformationModule
		Inherits SpreadsheetTutorialControl
		Public Sub New()
			InitializeComponent()
			RefreshPreview()
			SpreadsheetTutorialPanel.Dock = DockStyle.Fill
		End Sub

		Protected Overrides ReadOnly Property DefaultFileName() As String
			Get
				Return "EmployeeInformation"
			End Get
		End Property

		Public Overrides Function CreateBook() As IWorkbook
			InitializePreviewControl()
			Dim generator As New EmployeeInformationDocumentGenerator(DefaultCulture)
			Return generator.Generate(DemoUtils.GetRelativePath("EmployeeInformation_template.xlsx"))
		End Function

		Private Sub InitializePreviewControl()
			SpreadsheetTutorialPanel.Location = Point.Empty
		End Sub
	End Class
End Namespace
